src/windows/utils/filesystem.cpp

```diff
@@ -17,4 +17,12 @@ std::vector<std::string> hwinfo::filesystem::getDirectoryEntries(const std::stri
   return {};
 }
 
+void hwinfo::filesystem::get_specs_by_file_path(const std::string& path, int64_t& value) {
```

Comment:
This will never be needed nor implemented for windows. Thus, it can just be undefined for windows build